### DISCUSSION TITLE
Error: Minified React error #185

### DISCUSSION THREAD

**Discussion by DamonGuzman** [AntlerVC/firetable#1017] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**
I imported a collection

Something went wrong
Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

**Comment by bofeiw** [comment, marked as answer]
Can you add steps to re-create this error? It will help us investigate, thank you!

**Reply by BeeBombshell** [reply]
Hey @DamonGuzman, is this still an issue?

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @DamonGuzman if you're still facing this issue, feel free to create a new one.

**Reply by DamonGuzman** [reply]
Sorry, 

I just gave up on using Rowy since I didn't feel like debugging an issue for a trial software.
